src/components/MoviesSearchBar/MoviesSearchBar.js

```diff
@@ -6,9 +6,10 @@ import "./MoviesSearchBar.css";
 const MoviesSearchBar = () => {
   const [query, setQuery] = useState("");
   const dispatch = useDispatch();
+  const ENTER_KEY = 13;
```

Comment:
constants should be stored in a separate file and in separate folder (e.g. constants) because they will be used across the app

Reply:
I moved ENTER_KEY to the separate file in the constants directory - you will see the changes in the next PR